docs/architecture.md

```diff
@@ -73,10 +73,19 @@ The Cortex services are:
 - [Distributor](#distributor)
 - [Ingester](#ingester)
 - [Querier](#querier)
-- [Query frontend](#query-frontend) (optional)
-- [Ruler](#ruler) (optional)
 - [Alertmanager](#alertmanager) (optional)
 - [Configs API](#configs-api) (optional)
+- [Overrides exporter](./guides/overrides-exporter.md) (optional)
+- [Query frontend](#query-frontend) (optional)
+- [Query scheduler](./operations/scalable-query-frontend.md#query-scheduler) (optional)
+- [Ruler](#ruler) (optional)
+- [Compactor](./blocks-storage/compactor.md) (required for blocks storage)
+- [Store gateway](./blocks-storage/store-gateway.md) (required for blocks storage)
+- [Table manager](./chunks-storage/table-manager.md) (required for chunks storage)
+- [Flusher](./chunks-storage/ingesters-with-wal.md) (optional for chunks storage)
+- [Purger](./guides/deleting-series.md) (optional for chunks storage)
```

Comment:
Chunks storage is deprecated; I think we should be taking it out of docs not adding more references.

Reply:
Makes sense. I've dropped references to services used with chunks storage.